src/OpenLoco/Audio/Audio.h

```diff
@@ -99,10 +99,10 @@ namespace OpenLoco::Audio
     void pauseSound();
     void unpauseSound();
     void playSound(Vehicles::Vehicle2or6* t);
-    void playSound(sound_id id, Map::map_pos3 loc);
-    void playSound(sound_id id, Map::map_pos3 loc, int32_t pan);
+    void playSound(sound_id id, Map::Pos3 loc);
+    void playSound(sound_id id, Map::Pos3 loc, int32_t pan);
```

Comment:
whilst we are at it should we perhaps make these `const &`

Reply:
Yea, I'll do it later today.